### PR TITLE
Added in interactive matlab page, modifed the yml

### PR DESCRIPTION
**Pull request by Jeffotter**
Im not too confident in my matlab knowlegde if anyone would like to add anything else into this one

### CODE REVIEW

File: docs/_user_guide/matlab_interactive.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+```
+
+To access MATLAB's help documentation, type `doc` for the documentation browser or `help` followed by a function name.
+
+```title="Matlab Console"
+>> doc
+>> help disp
+```
+
+## Managing Matlab Software Libraries and Versions
```
Comment (natalya-patrikeeva) [inline docs/_user_guide/matlab_interactive.md:65]: that's not right. remove this paragraph

**Review by natalya-patrikeeva — CHANGES_REQUESTED** [review]

File: docs/_user_guide/matlab_interactive.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+```matlab title="hello_world.m"
+disp('Hello world')
+```
+
+Save this line to a new file called `hello_world.m`. 
+
+This one-liner script can be run on the matlab interactive Yens with 
+``` title="Terminal Input"
+matlab -nodesktop < hello_world.m
```
[the remark this answers is not in the document]
Comment (Jeffotter) [reply docs/_user_guide/matlab_interactive.md:90]: But this is for interactive yens?

[the remark this answers is not in the document]
Comment (Jeffotter) [reply docs/_user_guide/matlab_interactive.md:90]: Do we want to do JupyterHub in this?

**Review by natalya-patrikeeva — APPROVED** [review]
should be g2g

**Pull request merged**